scripts/constants.js

```diff
@@ -0,0 +1,22 @@
+const flags = [/myapp/g, /MyApp/g]
+const longNameFlags = [/My App/g]
+
+/* DIR PATHS */
+const UIPath = 'packages/ui/'
+const APIPath = 'packages/api/'
+
+const packageJson = 'package.json'
+const yamlFile = 'serverless.yaml'
```

Comment:
(nit) I'm not 100% sure about `yamlFile` because it isn't descriptive as our other constants